src/components/checkboxes/BasicCheckbox/BasicCheckbox.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC, InputHTMLAttributes } from 'react'
+
+import styles from './basicCheckbox.module.css'
+
+interface IClasses {
+   wrapper?: string
+   checkboxWrapper?: string
+   checkbox?: string
+   checkboxInterface?: string
+   checkboxLabel?: string
+}
+
+export interface BasicCheckboxCustomProps {
+   label?: string
+   classes?: IClasses
+}
+
+export type BasicCheckboxProps = BasicCheckboxCustomProps & InputHTMLAttributes<HTMLInputElement>
+
+const BasicCheckbox: FC<BasicCheckboxProps> = ({
+   label = '',
+   checked = false,
+   classes = { wrapper: '', checkbox: '', checkboxWrapper: '', checkboxInterface: '', checkboxLabel: '' },
+   ...props
+}) => {
+   const checkboxId = `checkbox-${Math.random().toString(36).substring(2, 11)}`
```

Comment:
Должна быть возможность задать свой `id` из пропсов. Если его нет, тогда использовать рандомный.
```suggestion
   const checkboxId = props.id ?? `checkbox-${Math.random().toString(36).substring(2, 11)}`
```